.github/workflows/tests.yml

```diff
@@ -45,3 +45,26 @@ jobs:
     - name: Test with pytest
       run: |
         make test
+  
```

Comment:
I think we can add `peft` as an dev/test dependency which is always installed in the test. then we don't need an extra peft CI (we already have 10 tests that run :)). wdyt?

Reply:
I proposed a hack to run the tests on the CI runners, however I think we should keep at least one runner that runs the tests without `peft` to check that everything works fine w/o `peft`